cli/ballerina-cli/src/test/resources/test-resources/projectsForDumpGraph/package_a/Dependencies.toml

```diff
@@ -0,0 +1,25 @@
+# AUTO-GENERATED FILE. DO NOT MODIFY.
+
+# This file is auto-generated by Ballerina for managing dependency versions.
+# It should not be modified by hand.
+
+[ballerina]
+dependencies-toml-version = "2"
```

Comment:
@azinneera For these particular tests do we need to commit the Dependencies.toml as well?